fuse/plugins/truth_information/peak_truth.py

```diff
@@ -23,11 +23,45 @@ class PeakTruth(strax.OverlapWindowPlugin):
     data_kind = "peaks"
 
     dtype = [
-        ("s1_photons_in_peak", np.int32),
-        ("s2_photons_in_peak", np.int32),
-        ("ap_photons_in_peak", np.int32),
-        ("pi_photons_in_peak", np.int32),
-        ("raw_area_truth", np.float32),
+        (("Number of photons from S1 scintillation in the peak.", "s1_photons_in_peak"), np.int32),
+        (("Number of photons from S2 scintillation in the peak.", "s2_photons_in_peak"), np.int32),
+        (("Number of photons from PMT afterpulses in the peak.", "ap_photons_in_peak"), np.int32),
+        (("Number of photons from photoionization in the peak.", "pi_photons_in_peak"), np.int32),
+        (
+            (
+                "Number of photoelectrons from S1 scintillation in the peak.",
+                "s1_photoelectrons_in_peak",
+            ),
+            np.int32,
+        ),
+        (
+            (
+                "Number of photoelectrons from S2 scintillation in the peak.",
+                "s2_photoelectrons_in_peak",
+            ),
+            np.int32,
+        ),
+        (
+            (
+                "Number of photoelectrons from PMT afterpulses in the peak.",
+                "ap_photoelectrons_in_peak",
+            ),
+            np.int32,
+        ),
+        (
+            (
+                "Number of photoelectrons from photoionization in the peak.",
+                "pi_photoelectrons_in_peak",
+            ),
+            np.int32,
+        ),
+        (
+            (
+                "Raw area from the scintillation, i.e. number of S1(S2) photoelectrons.",
+                "raw_area_truth",
```

Comment:
This PR reminds me these:

Though it is not this PR's problem, the naming `raw_area_truth` might not be most suitable. In WFSim, https://github.com/XENONnT/WFSim/blob/b75494fbd594c329eee54535b0bb9754bc7458d8/wfsim/strax_interface.py#L56 `raw_area` refers to the raw area, and is it float.

Reply:
I don't know what's the most appropriate naming for this quantity. But if anyone has better idea handling this I am 100% ok.